testsAT/src/test/resources/features/functionalAT/CCT_Installation_IT.feature

```diff
@@ -0,0 +1,102 @@
+@rest
+Feature: [QATM-1870] Service_Installation
+  Background: Setup DCOS-CLI
+    #Start SSH with DCOS-CLI
+    Given I open a ssh connection to '${DCOS_CLI_HOST}' with user 'root' and password 'stratio'
+
+  Scenario: [QATM-1870][01] Marathon-LB Installation - Retrive token of enviroment
+    When I open a ssh connection to '${BOOTSTRAP_IP}' with user '${ROOT_USER:-root}' and password '${ROOT_PASSWORD:-stratio}'
+    When I run 'cat /stratio_volume/vault_response |jq '.root_token'| tr -d "\""' in the ssh connection and save the value in environment variable 'root_token'
+    Then  I run 'echo !{root_token}' in the ssh connection
+
+  @web
+  Scenario: [QATM-1870][02] Marathon-LB Installation -Retrieve cookies
+    Given I set sso token using host '${CLUSTER_ID}.${CLUSTER_DOMAIN:-labs.stratio.com}' with user 'admin' and password '1234'
+    Given My app is running in '${CLUSTER_ID}.${CLUSTER_DOMAIN:-labs.stratio.com}:443'
+    #Login into the platform
+    When I securely browse to '#/login'
+    And I wait '5' seconds
+    When '1' elements exists with 'id:oauth-iframe'
+    And I switch to the iframe on index '0'
+    And '1' elements exists with 'xpath://*[@id="username"]'
+    And I type 'admin' on the element on index '0'
+    And '1' elements exists with 'xpath://*[@id="password"]'
+    And I type '1234' on the element on index '0'
+    And '1' elements exists with 'id:login-button'
+    And I click on the element on index '0'
+    And I wait '5' seconds
+    Given I securely send requests to '${CLUSTER_ID}.${CLUSTER_DOMAIN:-labs.stratio.com}:443'
+    When I securely browse to '/service/deploy-api/swagger-ui.html#!/'
+    Then I save selenium cookies in context
+    And I save selenium dcos acs auth cookie in variable 'DCOS_AUTH_COOKIE'
+    Then  I run 'echo !{DCOS_AUTH_COOKIE}' locally
+
+  Scenario: [QATM-1870][03] Marathon-LB Installation - Get versions installed in Command Center
+    Given I run 'curl -s -XGET -k -H "Cookie:dcos-acs-auth-cookie=!{DCOS_AUTH_COOKIE}" -H 'Content-Type: application/json' -H 'Accept: application/json' https://${CLUSTER_ID}.${CLUSTER_DOMAIN:-labs.stratio.com}/service/deploy-api/universe/postgres | awk 'FNR == 1'| tr -d "\"" | tr -d []' in the ssh connection and save the value in environment variable 'relese_name'
```

Comment:
Change postgres for **marathon-lb** 

https://${CLUSTER_ID}.${CLUSTER_DOMAIN:-labs.stratio.com}/service/deploy-api/universe/marathon-lb